lms/templates/main_django.html

```diff
@@ -27,30 +27,33 @@
 </head>
 
 <body class="{% block bodyclass %}{% endblock %} lang_{{LANGUAGE_CODE}}">
```

Comment:
Yeah, nothing you did here - just wayyyy too much logic and last minute "if its_flag_day {}, elsif its_christmas {}, elsif its_specialsnowflake" logic.